src/modules/XMLParser.ts

```diff
@@ -3,7 +3,14 @@ import * as saxes from 'saxes'
 import { RichTextElement } from '../types'
 import { get } from 'lodash'
 
+const ENTITIES = new Map([
+  ['"', '&quot;'],
+  ['&', '&amp;'],
+  ['<', '&lt;']
```

Comment:
Why is '>'  (&gt;) no problem here? (or other special characters)

Reply:
In XML attributes these are the only three problematic characters, that's the spec :)